### PR TITLE
Added functions mentioned below

### PR DESCRIPTION
**Pull request by chelsi-001011**
1)Added dropdown for yearwise view 
2)Completed HR list in company details view
   a)Added mail and call functionality to hr contacts
   b)Added random hr profile pic using robohash api

### CODE REVIEW

File: diary_portal/settings.py [hunk 1]
```diff
@@ -22,10 +22,9 @@
 # See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.environ.get('SECRET_KEY')
-
+SECRET_KEY = os.environ.get('SECRET_KEY');
```
Comment (sksingh1202) [inline diary_portal/settings.py:25]: There is no need for a semi-colon at the end of the assignment.

File: diary_portal/settings.py [hunk 2]
```diff
@@ -22,10 +22,9 @@
...
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = (os.environ.get('DEBUG') == 'True')
+DEBUG =(os.environ.get('DEBUG') == 'True');
```
Comment (sksingh1202) [inline diary_portal/settings.py:27]: Here too, there is no need for a semicolon at the end of this!

**Review by sksingh1202 — CHANGES_REQUESTED** [review]
Read the review comments and try removing both the errors!

**Review by sksingh1202 — APPROVED** [review]
Great job!

**Comment by chelsi-001011** [comment]
@sksingh1202 I have resolved the bug in another commit

**Pull request merged**